apis/core/v1alpha1/resource.go

```diff
@@ -197,9 +197,14 @@ type ResourceSpec struct {
 	// +optional
 	ClassReference *corev1.ObjectReference `json:"classRef,omitempty"`
 
+	// ProviderConfigReference specifies the provider that will be used to create,
+	// observe, update, and delete this managed resource.
+	ProviderConfigReference *Reference `json:"providerConfigRef,omitempty"`
```

Comment:
Do you think we could get away with something shorter like `pcRef` here? `providerConfigRef` is probably better - just thinking out loud.

Reply:
`pcRef` sounds a bit cryptic to me. I'd go with `providerConfigRef`.